syncer/syncer.go

```diff
@@ -1868,7 +1872,18 @@ func (s *Syncer) handleQueryEvent(ev *replication.QueryEvent, ec eventContext) e
 		return err
 	}
 
-	if s.cfg.ShardMode == "" {
+	// VIEW statements are not needed sharding synchronized, so pretend this is no-shard-mode
```

Comment:
What's the behavior when view statement in resharding sync? such as `ddl1 view dd2`. How about add a test for it.

Reply:
https://github.com/pingcap/dm/pull/1310/commits/56ba8266e01adbae8176a8f9b6dcc1c43f0ee3c4

in pessimistic mode, DDL lock will block sync of VIEW. in other cases VIEW is executed directly in downstream